modules/manage/pages/audit-logging.adoc

```diff
@@ -47,20 +47,20 @@ Some key steps to tuning your audit logging settings include:
 * Set `audit_enabled` to `true`.
 * Assess the retention needs for your audit logs. You may not need to keep the logs around for the default seven days. This is controlled by setting `retention.ms` for the `__audit_log` topic or by setting `delete_retention_ms` at the cluster level.
 
-The sequence of commands in rpk for these steps is as follows:
+Follow this sequence of commands in rpk for the steps described above:
```

Comment:
What about:
Follow this sequence of commands in rpk to configure audit logging: